### DISCUSSION TITLE
训练模型文件该如何选择？

### DISCUSSION THREAD

**Discussion by pcycccccc** [PaddlePaddle/PaddleOCR#13072] [Q&A]
我准备要训练英文数字的模型，选择了en_PP-OCRv3_det_slim_distill_train，下载下来发现里边有好几个训练的文件，不知道要如何选择，有没有人可以解答一下
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/108455796/62006b79-a048-4437-b701-eaf786bd3dc8)

**Comment by GreatV** [comment]
`._`可以删除，是mac系统产生的。best_recall.pdparams应该是你需要的文件
